mtest/Makefile

```diff
@@ -27,7 +27,7 @@ CURL = curl -fsL
 PLACEMAT_DATADIR = /var/scratch/placemat
 CT = /usr/local/bin/ct
 SUDO = sudo
-PACKAGES = libseccomp-dev automake autoconf libtool
+PACKAGES = libseccomp-dev automake autoconf libtool 
```

Comment:
Does this trailing space have some specific objective?

Reply:
Sorry, there was an extra space, so I fixed it.
5962af98f1653fcf77165fb8ecc11e5a4262ace6